website/docs/docs/api/persist.md

```diff
@@ -246,23 +268,88 @@ have persisted state based on a previous version of your store model. The user's
 persisted state may not align with that of your new store model, which could
 result in errors when a component tries to consume/update the store.
 
-Easy Peasy does its best to try and minimize / alleviate this risk. By default
-the persist API utilizes the `mergeDeep` strategy (you can read more above merge
-strategies further below). The `mergeDeep` strategy attempts to perform an
-optimistic merge of the persisted state against the store model. Where it finds
-that the persisted state is missing keys that are present in the store model, it
-will ensure to use the respective state from the store model. It will also
-verify the types of data at each key. If there is a misalignment (ignoring
-`null` or `undefined`) then it will opt for using the data from the store model
-instead as this generally indicates that the respective state has been
+By default the persist API utilizes the mergeDeep strategy (you can read more
+about merge strategies further below). The mergeDeep strategy attempts to
+perform an optimistic merge of the persisted state against the store model.
+Where it finds that the persisted state is missing keys that are present in the
+store model, it will ensure to use the respective state from the store model. It
+will also verify the types of data at each key. If there is a misalignment
+(ignoring null or undefined) then it will opt for using the data from the store
+model instead as this generally indicates that the respective state has been
```

Comment:
```suggestion
By default the persist API utilizes the `mergeDeep` strategy (you can read more
about merge strategies further below). The `mergeDeep` strategy attempts to
perform an optimistic merge of the persisted state against the store model.
Where it finds that the persisted state is missing keys that are present in the
store model, it will ensure to use the respective state from the store model. It
will also verify the types of data at each key. If there is a misalignment
(ignoring `null` or `undefined`) then it will opt for using the data from the store
model instead as this generally indicates that the respective state has been
```

I think we still want code highlights here?

Reply:
I think these were probably lost when I copied and pasted into the GH comment. The order I suggested was probably copied from there rather than re-ordered from the docs source itself. Looks good now in any case!

Reply:
Yup I had copied that over from the comment; all fixed now